### PR TITLE
[sf-move-subscriptions-api] adding riffraff build config and CI/CD for generating cfn templates from AWS CDK

### PR DESCRIPTION
**Pull request by mkuzdowicz**
## overview
[sf-move-subscriptions-api] adding riffraff build config and enabling generation of `cdk-cfn.yaml` file on teamcity build time

## what was done
- adding riffraff build config
- moving cdk directory to root to be able to build cfn templates from cdk on build time in CI/CD
- creating `generate_cfn_templates.sh` script which will build cfn templates from cdk on build time in CI/CD
- README instructions

## tests
- [x] CODE

### CODE REVIEW

File: handlers/sf-move-subscriptions-api/cfn.yaml [hunk 1]
```diff
@@ -0,0 +1,538 @@
+Parameters:
+  stage:
```
Comment (mkuzdowicz) [inline handlers/sf-move-subscriptions-api/cfn.yaml]: please ignore the cfn.yml file as it is autogenerated by cdk in handlers/sf-move-subscriptions-api/cdk/lib/sf-move-subscriptions-stack.ts file

Comment (twrichards) [reply handlers/sf-move-subscriptions-api/cfn.yaml]: please can we make it generate at build time then? so it's not in version control (in addition to the CDK code) but is still available for riff-raff purposes

Comment (mkuzdowicz) [reply handlers/sf-move-subscriptions-api/cfn.yaml]: i don't think that riffraff is supporting that at the moment

Comment (mkuzdowicz) [reply handlers/sf-move-subscriptions-api/cfn.yaml]: aa you mean in team city

Comment (mkuzdowicz) [reply handlers/sf-move-subscriptions-api/cfn.yaml]: that would be use-full

Comment (mkuzdowicz) [reply handlers/sf-move-subscriptions-api/cfn.yaml]: more less done

**Pull request merged**